__test__/Auth.spec.ts

```diff
@@ -31,6 +32,16 @@ const postAuthentication = async (credentials: { email?: string; password?: stri
   return await request(app).post('/api/1.0/auth').send(credentials);
 };
 
+const postLogout = async (options: { token?: string } = {}) => {
+  const agent = request(app).post('/api/1.0/logout');
+
+  if (options.token) {
+    agent.set('Authorization', `Bearer ${options.token}`);
```

Comment:
Will use the Auth Header to extract the user and use it to delete from the database